packages/react/src/hooks/contracts/useContractRead.ts

```diff
@@ -103,10 +122,10 @@ export function useContractRead({
   )
 
   const enabled = React.useMemo(() => {
-    let enabled = Boolean(enabled_ && addressOrName && functionName)
```

Comment:
How come we omitted functionName here?

Reply:
`functionName` is a required config option so seemed unnecessary. think we should include it?